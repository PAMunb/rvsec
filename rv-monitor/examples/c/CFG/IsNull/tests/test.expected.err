Test 1
isNull not called before deref!
isNull not called before deref!
Test 2
