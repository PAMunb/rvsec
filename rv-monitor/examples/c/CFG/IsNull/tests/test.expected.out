Test 1
sum: 15
Test 2
sum: 15
